runners/flink/src/test/java/org/apache/beam/runners/flink/FlinkStreamingTransformTranslatorsTest.java

```diff
@@ -65,12 +64,12 @@ public void readSourceTranslatorBoundedWithMaxParallelism() {
     env.setParallelism(parallelism);
     env.setMaxParallelism(maxParallelism);
 
-    StreamTransformation<?> sourceTransform =
-        applyReadSourceTransform(transform, PCollection.IsBounded.BOUNDED, env);
+    SourceTransformation<?> sourceTransform =
+        (SourceTransformation)
+            applyReadSourceTransform(transform, PCollection.IsBounded.BOUNDED, env);
```

Comment:
Why do we need to cast here, change the variable type?

Reply:
Because `SourceTransformation` and `OneInputTrasformation` no longer extend a common interface, so applyReadSourceTransform needs to return `Object` instead.